src/OrchardCore.Modules/OrchardCore.Email/Manifest.cs

```diff
@@ -5,6 +5,6 @@
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Provides the necessary infrastructure for configuring email settings and offers a default SMTP provider.",
+    Description = "Provides the necessary infrastructure for configuring email settings. Email providers can be enabled in separate features.",
```

Comment:
```suggestion
    Description = "Provides the necessary infrastructure for configuring email settings.",
```

Reply:
?

Reply:
I would just remove the comment about provided as it entails "how to" not description of the feature. I would just remove the old note about providing default SMTP provider

Reply:
I'd personally find it useful from the perspective of a newcomer, but you'll get a warning on the Email settings screens anyway.